api/client/src/main/java/jakarta/websocket/HandshakeResponse.java

```diff
@@ -34,7 +34,8 @@
     static final String SEC_WEBSOCKET_ACCEPT = "Sec-WebSocket-Accept";
 
     /**
-     * Return the list of Http headers sent by the web socket server.
+     * Return the map of Http headers to header values sent by the WebSocket server. The original case will be preserved
```

Comment:
Enforcing "original case" is tricky to implement for header field-names for many containers.

Also, what does it mean when there's multiple headers with mixed case?

```
X-Foo: bar
x-foo: baz
```